### ISSUE TITLE
training on svhn

### ISSUE THREAD

**Issue by ashimpd** [penny4860/YOLO-detector#16]
I tried using train_driver.py to train on svhn full training-data with following steps
1. generated annotations .xml files for each of the svhn image
2. used the from_scratch.json as config, with same anchors
3. Run it with 2 warmup and then 5 epochs, it doesn't give any boxes on the test image 1.png
4. If I use threshold as very low like 0.0001, then some boxes in wrong places are got.
5. I have used 10 labels 1 to 10 instead of just digits (updated in from_scratch.json as well)

Can you please tell me how have you trained your net that's present in the repo.
Do you think, it need to run for far more epochs, or should use only one label "digits"?

**Comment by penny4860** [comment]
@ashimpd  How many samples did you use in the training set?

**Comment by ashimpd** [comment]
I used all the samples present in svhn training set which is approx 33K

**Comment by penny4860** [comment]
@ashimpd  As a result of my experiments, it is difficult to learn multiple samples of svhn dataset at once. So I overfit a small number of samples (2 to 10) and then gradually add training samples. I am studying whether there is a better way of training.

**Comment by cyberh49** [comment]
@penny4860 Can you elaborate on how you trained the net and got good results? Thanks!

**Comment by penny4860** [comment]
@cyberh49 Thank you for your interest in this project. It is difficult to train several samples with the mobilenet structure. So I am doing resnet50-based experiments and I have confirmed that training is good for 1000 samples. This will soon be reflected in the master.

**Comment by penny4860** [comment]
@ashimpd @cyberh49 I updated to network based on resnet. I confirmed that training is converged for many samples (1000).
